### ISSUE TITLE
workspaceFolder path issue under Windows

### ISSUE THREAD

**Issue by eymeric-giraux** [useblocks/sphinx-needs-vscode#26]
Under Windows, it seems that the path using `${workspaceFolder}/` generates an issue.
See below.

![grafik](https://user-images.githubusercontent.com/122378774/235717427-3916d7f5-412d-461d-b9a1-efa2b89d7b48.png)

The problem is solved when using relative path.

**Comment by HughG** [comment]
Yep, I've just been hit by this, too.  To clarify the workaround by @eymeric-giraux, I solved the problem by using the following settings.

```json
{
    "sphinx-needs.needsJson": "build/needs/needs.json",
    "sphinx-needs.srcDir": "source"
}
```

*BUT* doing this makes the "Sphinx-Needs Explorer" view go "blank", whereas if I put

```json
{
    "sphinx-needs.needsJson": "${workspaceFolder}/build/needs/needs.json",
    "sphinx-needs.srcDir": "${workspaceFolder}/source"
}
```

then the explorer view works, but the above error appears and auto-complete, etc., don't work.
